src/main/java/edu/kis/powp/jobs2d/drivers/transformations/DriverLevelTransformation.java

```diff
@@ -0,0 +1,7 @@
+package edu.kis.powp.jobs2d.drivers.transformations;
+
+public interface DriverLevelTransformation {
```

Comment:
There is nothing concerning drivers in this interface. CoordsTransformation or XYTransformation would be better.